pkg/reconciler/utils/pubsub/topic_test.go

```diff
@@ -143,10 +142,6 @@ func newTestRunner(t *testing.T, tc testCase) (*testRunner, func()) {
 }
 
 func (r *testRunner) verify(t *testing.T, tc testCase, su *utilspubsubtesting.StatusUpdater, err error) {
-	if tc.wantErr != err {
-		t.Fatalf("Expect error, got: %v, want: %v", err, tc.wantErr)
```

Comment:
Should this be Errorf? Why does it need to be Fatalf?

Reply:
This is removed :)